src/pages/[platform]/how-amplify-works/faq/index.mdx

```diff
@@ -29,42 +29,43 @@ export function getStaticProps(context) {
+**Does Amplify Gen 2 support DataStore?**
 
-**Is Gen 2 backward compatible with existing Amplify projects on Gen 1?**
+Amplify code-first DX (Gen 2) supports GraphQL APIs without DataStore. We will release migration support for moving DataStore Gen 1 apps to Gen 2.
 
-No. Gen 2 apps adopt a file-based convention with a much smaller footprint. You cannot use the Gen 1 CLI to update the `resource.ts` files in Gen 2.
+**What programming languages does Amplify Gen 2 support?**
 
-**Is there a way to upgrade an existing Amplify project from Gen 1 to Gen 2?**
+Amplify Gen 2 supports a wide range of programming languages for client-side development. This includes dedicated client-side libraries for JavaScript, TypeScript, Dart, Java, Kotlin, and Swift. For backend development, Amplify Gen 2 uses TypeScript.
+
+**In Gen 1, Amplify offered a set of use case categories for building applications (for example, Authentication, Analytics, API, DataStore, Geo, and Predictions). Are those same categories available in Gen 2?**
 
-While Gen 2 is in public preview, we will provide migration guides that describe how to manually move from Gen 1 to Gen 2. When generally available, we will offer tooling that will help developers move projects from Gen 1 to Gen 2.
+Amplify code-first DX (Gen 2) supports Auth, Data, Storage, and Functions. Other use cases can be implemented using AWS Cloud Development Kit (AWS CDK) constructs which there is documentation for under the respective category name.
 
-**What platforms will Gen 2 support?** 
+**What if we want to add a feature like AI/ML or Amazon Location Service to our application in Gen 2?**
 
-While in public preview, Gen 2 supports a TypeScript-first developer experience for building backends. Teams that have built backends with Gen 2 can reuse that backend in mobile apps if desired.
+Because Amplify builds on the AWS CDK, any of the 175+ AWS services can be added to your app using [custom resources](/[platform]/build-a-backend/add-aws-services/custom-resources/) and L2/L1 AWS CDK constructs.
 
-**Which web frameworks will have first-class support?** 
+**What happens once my application grows too big and I want to do more configuration with my application (add more features, other AWS services, etc.)?*
 
-Gen 2 offers support for all web frameworks that are supported in the Amplify libraries (currently Next.js, React, Vue, and Angular).
+Amplify is built on top of the AWS CDK, so you can add any [AWS service to your Amplify app](https://docs.amplify.aws/gen2/build-a-backend/add-aws-services/custom-resources/). You can also override [Amplify-generated configuration of your resources](https://docs.amplify.aws/gen2/build-a-backend/add-aws-services/custom-resources/) using the CDK.
```

Comment:
Do we need to update these links? It looks like it's pointing to the `gen2` subpath which no longer exists

I think we could do something like this:
```suggestion
Amplify is built on top of the AWS CDK, so you can add any [AWS service to your Amplify app](/[platform]/build-a-backend/add-aws-services/custom-resources/). You can also override [Amplify-generated configuration of your resources](/[platform]/build-a-backend/add-aws-services/custom-resources/) using the CDK.
```

Reply:
Yep, good catch!